bip-0075.mediawiki

```diff
@@ -21,6 +21,12 @@ This BIP is an extension to BIP 70 that provides two enhancements to the existin
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and 
 "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
 
+==Copyright==
+
+<img src="https://licensebuttons.net/l/by/3.0/88x31.png">
+
+This BIP is licensed under CC BY.
```

Comment:
The version should be explicit (note that 4.0 is the only acceptable version).

Reply:
> The version should be explicit (note that 4.0 is the only acceptable version).